tests/test_pool.py

```diff
@@ -60,6 +60,70 @@ class TestCase(BaseTestCase):
         self.assertEqual(self.counter, self.max_size)
 
 
+class TestCaseClose(BaseTestCase):
+    max_size = 10
+
+    class Instance:
+        def __init__(self):
+            self.closed = False
+
+        async def close(self):
+            if self.closed:
+                raise RuntimeError
+
+            self.closed = True
+
+    def setUp(self):
+        self.instances = set()
+        super().setUp()
+
+    async def create_instance(self):
+        obj = TestCaseClose.Instanse()
+        self.instances.add(obj)
+        return obj
+
+    async def test_close(self):
+        async def getter():
+            async with self.pool.acquire() as instance:
+                assert instance > 0
```

Comment:
Constructor returns object of `Instance` class which doesn't have `__gt__` method.